content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+---
```

Comment:
```suggestion
author: >
  Richard Belleville (Google),
  Frank Budinsky (IBM),
  Arko Dasgupta (Tetrate),
  John Howard (Solo.io),
  Mattia Lavacca (Kong),
  Christine Kim (Isovalent),
  Grant Spence (Red Hat),
  Gina Yeh (Google),
  Candace Holman (Red Hat)
---
```

Reply:
should make it look like other posts
![image](https://github.com/kubernetes/website/assets/878251/43eef361-2535-4bc8-accb-cee0c472edb8)